fundamentals/fcsreqs.tex

```diff
@@ -26,6 +26,12 @@ \subsubsection{Discrete Facilities and Materials}
 
 \subsubsection{Open Access}
 
+% gonuke: I am perhaps oversensitive, but I perceive the status of 
+%         VISION to be constantly shifting.  I am thus unsure whether
+%         ``license agreemnts or instutional approval'' are currently
+%         required for access to VISION.  I raise it here to satisfy
+%         my desire to have it categorically confirmed by someone.
```

Comment:
This is a good point. I think at this point, you do still have to ask them for it, but you may not have to do too much paperwork. Ultimately though, it relies on powersim, which I think does require a significant license, doesn't it?